### PR TITLE
created html templates to match the current site style

### PR DESCRIPTION
**Pull request by hanlray**

### CODE REVIEW

**Comment by tzach** [comment]
Thanks @hanlray I will take a look 
Question: does it make sense to link to scylladb.com css instead of copy it to this project?
If we want to update only the doc look it make sense.
@haimagami @sagish can you share your thoughts?
If you can review this PR for the new doc site it will be appreciated

**Comment by tzach** [comment]
@hanlray are the instructions for running the site on Mac in the README are correct?
Can you give some inputs on you run this site locally?

**Comment by hanlray** [comment]
I tested on Debian and Windows and fixed a small issue in _utils/setup.sh

**Comment by hanlray** [comment]
yes, It makes sense. We can use a separate css dedicated for the doc site
if necessary.
I'll change this.

**Comment by hanlray** [comment]
@tzach changed to the assets(css, js and images) from www.scylladb.com instead of the local copy, check please

File: _utils/setup.sh [hunk 1]
```diff
@@ -5,10 +5,10 @@ if pwd | egrep -q '\s'; then
...
-which virtualenv || (
-	which brew && brew install python && pip install virtualenv
-	which dnf && sudo dnf install python-virtualenv
-)
+which virtualenv\
+||which brew && brew install python && pip install virtualenv\
+||which dnf && sudo dnf install python-virtualenv\
+||which apt-get && sudo apt-get install python-virtualenv
 
 which virtualenv || echo "Failed to find or install virtualenv."
```
Comment (tzach) [inline _utils/setup.sh]: @hanlray this change break the make for me on Fedora 24

before the change:
```
$ make
./_utils/setup.sh
/usr/bin/virtualenv
/usr/bin/virtualenv
New python executable in /home/tzach/projects/scylla-docs/bin/python
Installing setuptools, pip, wheel...done.

....

preparing documents... done
writing output... [100%] lib64/python2.7/site-packages/wheel-0.30.0a0.dist-info/DESCRIPTION                                                                                              
generating indices... genindex
writing additional pages... search
copying static files... WARNING: html_static_path entry u'/home/tzach/projects/scylla-docs/_static' does not exist
done
copying extra files... done
dumping search index in English (code: en) ... done
dumping object inventory... done
build succeeded, 82 warnings.

Build finished. The HTML pages are in _build/dirhtml.
```
after the change

```
$ make
./_utils/setup.sh
/usr/bin/virtualenv
./_utils/setup.sh: line 9: brew: command not found
/usr/bin/dnf
Failed to synchronize cache for repo 'scylla', disabling.
Failed to synchronize cache for repo 'scylla-generic', disabling.
Failed to synchronize cache for repo 'cloudrouter', disabling.
Last metadata expiration check: 0:18:39 ago on Sun Nov 27 11:35:46 2016.
Dependencies resolved.
=========================================================================================================================================================================================
 Package                                            Arch                                   Version                                         Repository                               Size
=========================================================================================================================================================================================
Installing:
 python2-virtualenv                                 noarch                                 15.0.3-2.fc24                                   updates                                 1.7 M

Transaction Summary
=========================================================================================================================================================================================
Install  1 Package

Total download size: 1.7 M
Installed size: 2.0 M
Is this ok [y/N]: y
Downloading Packages:
python2-virtualenv-15.0.3-2.fc24.noarch.rpm                                                                                                              2.8 MB/s | 1.7 MB     00:00    
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Total                                                                                                                                                    763 kB/s | 1.7 MB     00:02     
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Installing  : python2-virtualenv-15.0.3-2.fc24.noarch                                                                                                                              1/1 
  Verifying   : python2-virtualenv-15.0.3-2.fc24.noarch                                                                                                                              1/1 

Installed:
  python2-virtualenv.noarch 15.0.3-2.fc24                                                                                                                                                

Complete!
sudo: apt-get: command not found
Makefile:26: recipe for target 'bin/sphinx-build' failed
make: *** [bin/sphinx-build] Error 1
```

**Comment by tzach** [comment]
@hanlray please send pull requests from a branch not from your master.
It will be easier for you to iterate and move between pull requests
https://guides.github.com/introduction/flow/

**Pull request merged**